pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md

```diff
@@ -23,9 +23,10 @@ Before starting the Konvoy installation, verify that you have:
 ## Configure GCP prerequisites
 
 1.  Log in to GCP:
-<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    <!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+
     ```bash
-    gcp login
+    gcloud login
```

Comment:
The command would appear to be 'gcloud auth login':   https://cloud.google.com/sdk/gcloud/reference/auth/login